frontend/src/features/browser-profiles/profile-browser.ts

```diff
@@ -373,29 +387,7 @@ export class ProfileBrowser extends TailwindElement {
     this.iframeSrc = undefined;
     this.isIframeLoaded = false;
 
-    try {
-      await this.checkBrowserStatus();
-
-      this.browserNotAvailable = false;
```

Comment:
This was no longer throwing, instead moved the true/false change into checkBrowserStatus() itself